# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/gc/moz.build
# Commit: b1117df3b91d
# Full Hash: b1117df3b91d10d68d10b399637cf9ac295057e6
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2019-11-01 16:13:11
# Regressor Bug: 1587096
# File Overlap Count: 1
# Description:
#   Bug 1587096 - Part 3: Implmement FinalizationGroup r=sfink
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49946
# ==============================================================================

diff -r cefc0474e13c -r b1117df3b91d js/src/gc/moz.build
--- a/js/src/gc/moz.build	Fri Nov 01 10:37:40 2019 +0000
+++ b/js/src/gc/moz.build	Fri Nov 01 10:37:47 2019 +0000
@@ -26,6 +26,7 @@
     'Allocator.cpp',
     'AtomMarking.cpp',
     'Barrier.cpp',
+    'FinalizationGroup.cpp',
     'GC.cpp',
     'GCTrace.cpp',
     'Marking.cpp',